src/root_finding/krawczyk.jl

```diff
@@ -79,18 +79,20 @@ function krawczyk{T}(f::Function, f_prime::Function, x::Interval{T}, level::Int=
 
     debug && @show(x,m)
 
-    isthin(x) && return [(x, :unknown)]
+    isthin(x) && return [Root(x, :unknown)]
 
     # bisecting
     roots = vcat(
         krawczyk(f, f_prime, Interval(x.lo, m), level+1,
                  tolerance=tolerance, debug=debug, maxlevel=maxlevel),
-        krawczyk(f, f_prime, Interval(nextfloat(m), x.hi), level+1,
+        krawczyk(f, f_prime, Interval(m, x.hi), level+1,
```

Comment:
I recall that we deliberately decided to use `nextfloat(m)` to make sure that the two intervals considered were somehow separated. (Same comment for newton.jl.) Can you explain the reason behind the change?